src/main/java/ch/njol/skript/expressions/ExprAltitude.java

```diff
@@ -37,5 +38,14 @@ protected String getPropertyName() {
 	public Class<? extends Number> getReturnType() {
 		return Number.class;
 	}
-	
+
+	@Override
+	public Expression<? extends Number> simplify() {
+		// as of INSERT VERSION, there are no literal locations but this is implemented for when pure functions can
```

Comment:
just want to note that the insert version script won't cover cases like this

Reply:
Are you sure? It seems indiscriminate to me: `find src/main/java -type f -exec sed -i "s/INSERT VERSION/$1/g" {} \;`